config.yml

```diff
@@ -7,6 +7,7 @@ CMS_OAUTH2_SECRET: 2icx54cdu9SJ77KMlZgHTWcm
 CONTACT_EMAIL: noreply@community.abzt.de
 DOCKER_IMAGE_CADDY: docker.io/library/caddy:2.7.5
 DOCKER_IMAGE_OPENEDX: docker.io/abstract2tech/community-theme-openedx:17.0.0
+EDX_PLATFORM_VERSION: open-release/quince.3
```

Comment:
I have an impression that tutor 17.0.4 does that out of the box

Reply:
Yes but I did that, because may be the runner if it's not updated to latest tutor version, might override it